src/GameLogic/PlayerActions/Character/IncreaseStatsAction.cs

```diff
@@ -24,16 +24,19 @@ public void IncreaseStats(Player player, AttributeDefinition statAttributeDefini
             throw new InvalidOperationException("No character selected");
         }
 
-        if (player.SelectedCharacter.LevelUpPoints > 0)
+        var selectedCharacter = player.SelectedCharacter;
+        if (selectedCharacter.CharacterStatus != CharacterStatus.GameMaster && player.SelectedCharacter?.LevelUpPoints <= 0)
```

Comment:
Maybe introduce an extension method for a character, which determines, if a player can add points. So that the same check can be used in the AddStatChatCommandPlugIn

Reply:
Fixed

Reply:
Yeah, good call. Done